Livecheckables/dps8m.rb

```diff
@@ -0,0 +1,4 @@
+class Dps8m
+  livecheck :url   => "https://gitlab.com/dps8m/dps8m.git",
+            :regex => /^R(\d+(?:\.\d+)+)$/
```

Comment:
```suggestion
            :regex => /^R?(\d+(?:\.\d+)+)$/i
```

Since the regex only matches stable versions, we can make the leading `R` optional (matching versions like `1.2.3` and `R1.2.3`) without worrying about matching unstable versions.

It also wouldn't hurt to make this case-insensitive (appending `i` to the end of the regex), just to be on the safe side if they switch to using `r1.2.3` at any point.